README.MD

```diff
@@ -4,7 +4,7 @@
 [![Coverage](https://next.sonarqube.com/sonarqube/api/project_badges/measure?project=sonarlint-visualstudio&metric=coverage)](https://next.sonarqube.com/sonarqube/component_measures/domain/Coverage?id=sonarlint-visualstudio)
 
 
-SonarLint is a Visual Studio 2015, 2017 and 2019 extension that provides on-the-fly feedback to developers on new bugs and
+SonarLint is a Visual Studio 2017, 2019 and 2022 extension that provides on-the-fly feedback to developers on new bugs and
```

Comment:
Thank you for the contribution @MrCull, we appreciate the heads up. 
Our marketing team took a look and has plans to revamp the contents of this page, along with a deprecation message of SonarLint for Visual Studio 2017. We will decline this PR and implement their changes.
Thank you.